ios/RNCWebView.m

```diff
@@ -273,7 +273,7 @@ - (void)setupConfiguration:(RNCWebView*)sender {
         if ([sender.contentRuleLists containsObject:identifier]) {
           [contentRuleListStore lookUpContentRuleListForIdentifier:identifier completionHandler:^(WKContentRuleList *contentRuleList, NSError *error) {
             if (!error) {
-                [sender->wkWebViewConfig.userContentController addContentRuleList:contentRuleList];
+                [wkWebViewConfig.userContentController addContentRuleList:contentRuleList];
```

Comment:
In normal case, sender is current webview
but In case of window.open, sender is parent webview, so we have to remove it